packages/e2e-tests/test-applications/nextjs-13-app-dir/start-event-proxy.ts

```diff
@@ -0,0 +1,6 @@
+import { startEventProxyServer, waitForTransaction } from '../../test-utils/event-proxy-server';
+
+startEventProxyServer({
+  port: 27496,
```

Comment:
are these ports random? Should they not be?

Reply:
It's kinda tricky. For Next.js we need to know the port of the proxy server at runtime AND at build time. Playwright (and implicitly the proxy server) are started after the project is built (obviously).

Instead of writing some convoluted logic that picks a port and then passes it everywhere, for now, I just opted to pick _some_ port statically. Of course, this is somewhat error-prone but usually, it is just a "write once and forget" kinda thing.